src/controllers/expense.controller.js

```diff
@@ -0,0 +1,92 @@
+const expenseService = require('../services/expense.service.js');
+const userService = require('../services/user.service.js');
+
+const get = async (req, res) => {
+  const { categories, userId, from, to } = req.query;
+
+  const result = await expenseService.getAll(categories, userId, from, to);
+
+  return res.send(result);
+};
+
+const getById = async (req, res) => {
+  const { id } = req.params;
+
+  const expense = await expenseService.getById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const { userId, title, amount, category, note, spentAt } = req.body;
+  const existedUser = await userService.getById(userId);
+
+  if (!userId || !title || !amount || !spentAt || !existedUser) {
+    return res.sendStatus(400);
+  }
```

Comment:
The validation logic here is good, but it might be beneficial to include more detailed error messages to help the client understand why the request failed. Consider sending a JSON response with a message indicating which field is missing or invalid.